iphone/Classes/TiUIWebView.m

```diff
@@ -542,7 +547,10 @@ -(void)setHandlePlatformUrl_:(id)arg
 
 -(void)setUserAgent_:(id)value
 {
-    ENSURE_TYPE(value, NSString);
+    ENSURE_TYPE_OR_NIL(value, NSString);
+    if (value == nil || [value isEqualToString:@""]) {
+        value = defaultUserAgent;
```

Comment:
Here is the problem I had thinking more about this:

- This whole config will set it globally, not only for this webview
- We would need to store the "old" default user agent somewhere so we can restore it